src/external-modules/MemoryCard/MemoryCard.csproj

```diff
@@ -1,8 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
```HuaweiCloud.SDK.OBS``` only support ```net20```/```net35```/```net40```/```net45```/```netcoreapp2.0```, it will redirect to ```netcoreapp2.0``` when we target framework to ```netcoreapp3.1```

Reply:
目前Grpc 使用的是netcoreapp2.1, 等功能开发完成后，可以考虑降级为2.0，MemoryCard的基础存储建议使用MemoryCache，第三方存储可以单独出来